lib/MusicBrainz/Server/Data/URL.pm

```diff
@@ -106,6 +107,7 @@ my %URL_SPECIALIZATIONS = (
     'LyricEvesta'         => qr{^https?://lyric\.evesta\.jp/}i,
     'MainlyNorfolk'       => qr{^https?://(?:www\.)?mainlynorfolk\.info/}i,
     'Maniadb'             => qr{^https?://(?:www\.)?maniadb\.com/}i,
+    'Melon'               => qr{^https?://www\.melon\.com/}i,
```

Comment:
In case someone is unintentionally copying the part `melon.com/…` only, we still want it to match as a Melon URL. So I suggest following the general scheme here:

```suggestion
    'Melon'               => qr{^https?://(?:www\.)?melon\.com/}i,
```